netbox/resource_netbox_device_interface.go

```diff
@@ -0,0 +1,259 @@
+package netbox
+
+import (
+	"context"
+	"regexp"
+	"strconv"
+
+	"github.com/fbreckle/go-netbox/netbox/client"
+	"github.com/fbreckle/go-netbox/netbox/client/dcim"
+	"github.com/fbreckle/go-netbox/netbox/models"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/diag"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/validation"
+)
+
+func resourceNetboxDeviceInterface() *schema.Resource {
+	validModes := []string{"access", "tagged", "tagged-all"}
+
+	return &schema.Resource{
+		CreateContext: resourceNetboxDeviceInterfaceCreate,
+		ReadContext:   resourceNetboxDeviceInterfaceRead,
+		UpdateContext: resourceNetboxDeviceInterfaceUpdate,
+		DeleteContext: resourceNetboxDeviceInterfaceDelete,
+
+		Description: `:meta:subcategory:Device:From the [official documentation](https://docs.netbox.dev/en/stable/features/device/#interface):
```

Comment:
The subcategory should be "Data Center Inventory Management (DCIM)"

Reply:
thank you, has been resolved

Reply:
I meant to use literally `Data Center Inventory Management (DCIM)`. That string will be used on the website for the subcategories. I fixed it myself in a commit here.